src/instantiate.jl

```diff
@@ -69,6 +131,18 @@ function newpage(; path="page", overwrite=false)
             chmod(joinpath(root, file), 0o644)
         end
     end
+    # Try filling `config.md`
+    name = "config.md"
+    src = joinpath(store, name)
+    config = read(src, String)
+    dst = joinpath(path, name)
+    try 
+        config = replace(config, r"{{ ([a-z_]+) }}" => x->fill(Val(Symbol(x[4:end-3]))))
+        write(dst, config)
+    catch e
+        cp(src, dst, force=false)
+    end
```

Comment:
I would suggest just remove the try catch here (I'm not sure I understand its purpose)

Reply:
I'm not sure either. I thought it might fail but then I wrote everything like it shouldn't fail :smile: